services/web-app/components/page-tabs/page-tabs.js

```diff
@@ -4,58 +4,68 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import { Column, Grid, Tab, TabList, Tabs } from '@carbon/react'
+import { Column, Grid } from '@carbon/react'
 import clsx from 'clsx'
 import { useRouter } from 'next/router'
 import PropTypes from 'prop-types'
 
+import NextLink from '../next-link'
 import styles from './page-tabs.module.scss'
 
-const createTabs = ({ tabs }) => {
-  return tabs.map((tab, i) => {
-    return (
-      <Tab as="span" key={i}>
-        <a href={tab.path} className={styles['tab-link']}>
-          {tab.name}
-        </a>
-      </Tab>
-    )
-  })
-}
-
-const getPathLeaf = (fullPath) => {
-  // Get the last piece of the path without the query string
-  return fullPath.split('/').pop().split('?')[0].split('#')[0]
-}
-
-const PageTabs = ({ className, tabs = [] }) => {
+const PageTabs = ({ title, tabs, className }) => {
   const router = useRouter()
 
+  const getPathLeaf = (fullPath) => {
+    // Get the last piece of the path without the query string
+    return fullPath.split('/').pop().split('?')[0].split('#')[0]
+  }
+
```

Comment:
Since this is a pure function, moving this outside of the `PageTabs` component (so it's just a top-level function) is a slight performance boost since it means this function doesn't need to be re-evaluated every time this component re-renders.

Reply:
Done!